index.d.ts

```diff
@@ -24,6 +24,13 @@ declare namespace nested {
      * to preserve them.
      */
     preserveEmpty?: boolean
+
+    /**
+     * The plugin supports the SCSS custom at-rule `@at-root` which
+     * rule blocks out of the nesting. If you want you can choose a
+     * new custom name for this rule.
+     */
+    rootRuleName?: string
```

Comment:
For me, this seems like a sensible defence mechanism. 
`@at-root` is a wholly made up magic rule, and at some point it might be sensible to be able to rename it to avoid problems.
Whatever, it's an extremely inexpensive feature

Reply:
this is flagged as "outdated" as I did some rewording of the JSDoc comment.